app/models/enterprise.rb

```diff
@@ -30,6 +30,7 @@ class Enterprise < ApplicationRecord
   preference :shopfront_product_sorting_method, :string, default: "by_category"
   preference :invoice_order_by_supplier, :boolean, default: false
   preference :product_low_stock_display, :boolean, default: false
+  preference :hide_ofn_navigation, :boolean, default: false
```

Comment:
This seems to be a leftover from the old branch. You got an attribute on the enterprise now and you don't need this, right?

Reply:
Good catch, commit removed.